README.md

```diff
@@ -0,0 +1,21 @@
+# Cloudskiff
```

Comment:
Should we replace Cloudskiff occurences by `driftctl`?

Reply:
Pas bete